include/faker-cxx/Book.h

```diff
@@ -61,5 +61,68 @@ class Book
      * @endcode
      */
     static std::string isbn();
+
+	/**
+	 * @brief Returns a random release year
+	 *
+	 * @returns int year
+	 *
+	 * @code
+	 * Book::releaseYear() // "2016-06-27"
+	 * @endcode
+	 */
+	static std::string releaseYear();
```

Comment:
Please change to int, just 1996 for example

Reply:
Sure